package.json

```diff
@@ -25,6 +25,7 @@
     "cookie-parser": "^1.4.5",
     "express": "^4.17.1",
     "express-sslify": "^1.2.0",
+    "jquery": "^3.5.1",
```

Comment:
We shouldn't need to add jquery to this project just to get video/audio controls. It looks like the only use for it is some DOM element class manipulation below. Everywhere else we've done that with vanilla JS

Reply:
done